Bottom: 9c5a825d4a629e9971a993e6f93db66b41850d36
Top:    c0f2387d37f8fb13a0d133bb378e1187cf4b8e43
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 13:57:52 +0000

Disable polygon arc radius bodgering


---

diff --git a/src/polygon.h b/src/polygon.h
index f1af9bf..eb9ee15d 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -37,11 +37,7 @@
 #define POLY_CIRC_SEGS 10 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
-/* adjustment to make the segments outline the circle rather than connect
- * points on the circle: 1 - cos (\alpha / 2) < (\alpha / 2) ^ 2 / 2
- */
-#define POLY_CIRC_RADIUS_ADJ (1.0 + M_PI / POLY_CIRC_SEGS_F * \
-                                    M_PI / POLY_CIRC_SEGS_F / 2.0)
+#define POLY_CIRC_RADIUS_ADJ 1.0
 
 /* polygon diverges from modelled arc no more than MAX_ARC_DEVIATION * thick */
 #define POLY_ARC_MAX_DEVIATION 0.02
